ui/apps/akasha/src/extensions/beam-editor/beam-editor.tsx

```diff
@@ -277,13 +275,10 @@ export const BeamEditor: React.FC = () => {
                   }}
                   isNsfwCheckboxSelected={!!nsfwBlocks.get(idx)}
                 />
-                <ContentBlockExtension
-                  editMode={{
-                    appName: block.appName,
-                    propertyType: block.propertyType,
-                    externalHandler: value => updateBlockDisablePublishState(value, block.order),
-                  }}
-                  mode={ContentBlockModes.EDIT}
+                <EditContentBlockExtension
```

Comment:
I think we should rename this components to EditorBlockExtension and ContentBlockExtension for better understanding their purpose